### DISCUSSION TITLE
C2C menu blur effect

### DISCUSSION THREAD

**Discussion by emoose** [emoose/OutRun2006Tweaks#234] [Suggestions]
### Describe the feature you'd like to see added

C2C Xbox has a blur effect on the menus when a dialog is in front:

![xemu_2024-09-24_18-45-36](https://github.com/user-attachments/assets/2a2f5a9a-562b-4c18-81c9-30b33a8764b5)

On PC this is missing (TODO: check PS2)
![OR2006C2C_2024-09-24_18-45-39](https://github.com/user-attachments/assets/9edc5179-f913-4aa6-80a8-33a495d63400)

Possible they used the same blur code that bloom uses, which is broken on PC, so ended up leaving it disabled instead.

Since it's on the C2C side it would be hard to find the code changed for it though (hardly any C2C-specific code is mapped out atm, still crossing my fingers for a C2C prototype to show up one day with debug symbols 🤞)

**Comment by Tenome** [comment]
PS2 doesn't have a blur effect.

![image](https://github.com/user-attachments/assets/5e6bfb61-0528-40a6-b001-a5e1c20205e3)
